getnacelle/nacelle-js
# feat: add shopify-cart to nuxt reference store

### WHY are these changes introduced?

Fixes [[ENG-7067](https://nacelle.atlassian.net/browse/ENG-7067)] 



### WHAT is this pull request doing?

Adds shopify cart to the nuxt reference store
Replace checkout with shopify cart checkout


### How to Test

- `git checkout ENG-7067--shopify-cart-nuxt-reference-store`
- to the nuxt reference store project `.env` add the following: (reach out if you want the vars I used)

```
SHOPIFY_SHOP_ID="<YOUR_SHOPIFY_ID>"
SHOPIFY_STOREFRONT_ACCESS_TOKEN="<YOUR_SHOPIFY_STOREFRONT_ACCESS_TOKEN>"
```

- `cd reference-stores/nuxt`
- `npm i && npm run generate && npm run start`
- see if cart/checkout works
